thermalnetwork/ground_heat_exchanger.py

```diff
@@ -17,109 +17,223 @@
-            roughness=self.json_data["pipe"]["roughness"],
-            conductivity=self.json_data["pipe"]["conductivity"],
-            rho_cp=self.json_data["pipe"]["rho_cp"],
-        )
-        ghe.set_soil(
-            conductivity=self.json_data["soil"]["conductivity"],
-            rho_cp=self.json_data["soil"]["rho_cp"],
-            undisturbed_temp=self.json_data["soil"]["undisturbed_temp"],
-        )
-        ghe.set_grout(conductivity=self.json_data["grout"]["conductivity"], rho_cp=self.json_data["grout"]["rho_cp"])
-        ghe.set_fluid()
-        ghe.set_borehole(
-            height=self.json_data["geometric_constraints"]["max_height"],
-            # Assuming max height is the height of the borehole
-            buried_depth=self.json_data["borehole"]["buried_depth"],
-            diameter=self.json_data["borehole"]["diameter"],
-        )
-        ghe.set_simulation_parameters(
-            num_months=self.json_data["simulation"]["num_months"],
-            max_eft=self.json_data["design"]["max_eft"],
-            min_eft=self.json_data["design"]["min_eft"],
-            max_height=self.json_data["geometric_constraints"]["max_height"],
-            min_height=self.json_data["geometric_constraints"]["min_height"],
-            continue_if_design_unmet=True,
-            max_boreholes=2500,
-        )
-        ghe.set_ground_loads_from_hourly_list(self.json_data["loads"]["ground_loads"])
-        if "polygons" in self.json_data["geometric_constraints"]:
-            ghe.set_geometry_constraints_bi_rectangle_constrained(
-                property_boundary=self.json_data["geometric_constraints"]["polygons"][0],
-                no_go_boundaries=self.json_data["geometric_constraints"]["polygons"][1:],
-                b_min=self.json_data["geometric_constraints"]["b_min"],
-                b_max_x=self.json_data["geometric_constraints"]["b_max"],
-                b_max_y=self.json_data["geometric_constraints"]["b_max"],
+        self.json_data = data["properties"]
+        self.design_method = GHEDesignType[self.json_data["borefield"]["design_method"].upper()]
+        self.area = self.compute_area()
+
+        # # compute Area
+        # self.id = data["id"]
+        # if "polygons" in self.json_data["geometric_constraints"]:
+        #     bound_area = polygon_area(self.json_data["geometric_constraints"]["polygons"][0])
+        #     if bound_area < 0:  # clockwise polygon; reverse for GHE Designer
+        #         self.json_data["geometric_constraints"]["polygons"][0].reverse()
+        #     self.area = abs(bound_area)
+        #     for i, hole_poly in enumerate(self.json_data["geometric_constraints"]["polygons"][1:]):
+        #         hole_area = polygon_area(hole_poly)
+        #         if hole_area < 0:  # clockwise polygon; reverse for GHE Designer
+        #             self.json_data["geometric_constraints"]["polygons"][i + 1].reverse()
+        #         self.area -= abs(hole_area)
```

Comment:
@chriswmackey here's another FYI.

This PR no longer supports the lat/lon to coordinate translations you included, so this was commented out for now. I think we're going to move ahead and merge this PR (since it's already so big), but we need to follow up with a subsequent PR to add this back, as well as include the other GHE design methods in.

Reply:
@chriswmackey that would be fantastic if you could. I mostly wasn't sure what was going to work for you, so I skipped it just to keep things moving. If you could take the first pass and suggest something that works for you, we can help get it wrapped up and integrated in short order.

Reply:
I agree with @mitchute . Once our integration tests are finished I'll merge this - planning on this afternoon.

I think if you branch off this one, it'll automatically adjust once this is merged to main.